builder/plugins/custom-markdown-parse.js

```diff
@@ -1,6 +1,10 @@
-var marked = require('marked');
+const marked = require('marked');
+const { EOL } = require('os');
+
+
+function extractHeaders(content) {
+  const str = content.replace(/\r\n/g, '\n');
```

Comment:
maybe EOL here?

Reply:
I tried, but EOL didn't work in regexp for me.